PublishParasoftTestResults/package-lock.json

```diff
@@ -9,14 +9,14 @@
       "version": "1.0.7",
       "license": "Apache-2.0",
       "dependencies": {
-        "azure-pipelines-task-lib": "^2.7.7",
+        "azure-pipelines-task-lib": "^4.3.1",
         "dot-properties": "^1.0.1",
         "fs": "0.0.1-security",
         "sax": "^1.2.4",
         "xslt3": "^2.5.0"
       },
       "devDependencies": {
-        "@types/node": "^8.10.66",
+        "@types/node": "^16.18.23",
```

Comment:
Did you run npm install?
For me if I run npm install there are more changes.
For example, line 25 should have consistent node version for 16.18.23.